src/arguments/bricks.go

```diff
@@ -0,0 +1,76 @@
+package arguments
+
+import (
+	"io/fs"
+	"io/ioutil"
+	"log"
+	"path/filepath"
+	"strings"
+)
+
+type Brick struct {
+	// The brick's name. Usually the name of the parent directory
+	Name string
+	// The absolute path of the brick's directory
+	Path string
+	// The absolute path of the `brick.yml` file
+	ConfigurationFilePath string
+	// The content of the `brick.yml` to be parsed later on
+	ConfigurationFileContent []byte
+}
+
+// Walks the file system from the provided root, gathers all folders containing a `brick.html` file, and build a Brick struct from it.
+// Returns a pointer to a slice of Bricks.
+func GetBricksAndContent(root string) (*[]Brick, error) {
+	brickFiles := []Brick{}
+
+	err := filepath.WalkDir(
+		root,
+		func(path string, d fs.DirEntry, err error) error {
+			if d.Type().IsRegular() && d.Name() == "brick.yml" {
+				content, err := ioutil.ReadFile(path)
```

Comment:
*Arguably* useful since the go-yaml lib's `Marshall` and `Unmarshall` functions can handle `[]byte` and `*[]byte` inputs.

If we save the content in-memory once on startup, it might make it easier to work with the bricks down the line. Arugably